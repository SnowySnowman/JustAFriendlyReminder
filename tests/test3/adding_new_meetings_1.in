2
weekly taco briefings with Star
05/09/23
10:30
y
0001
6
