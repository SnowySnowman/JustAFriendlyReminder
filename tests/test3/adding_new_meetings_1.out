Just a friendly reminder! You have these tasks to finish today.

These tasks need to be finished in the next three days!

You have the following meetings today!
- Game Night with Star and Friends. at 19:00

You have the following meetings scheduled over the next week!
- Training Session with Sensei. on 07/09/23 at 09:00
- Movie Night at the Diaz Residence. on 10/09/23 at 20:30
- Parent-Teacher Conference at Echo Creek High. on 08/09/23 at 16:30
- Visit to Abuelita's House. on 12/09/23 at 13:00

What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
Please enter a meeting description:
Please enter a date:
Please enter a time:
Ok, I have added weekly taco briefings with Star on 05/09/23 at 10:30.
Would you like to share this meeting? [y/n]: Who would you like to share with?
0001 Jackie
0003 Star
0004 Tom
Meeting shared.
What would you like to do?
1. Complete tasks
2. Add a new meeting.
3. Share a task.
4. Share a meeting.
5. Change Jafr's master directory.
6. Exit
